cloudevents/http/__init__.py

```diff
@@ -15,6 +15,7 @@
 import typing
 
 from cloudevents.http.event import CloudEvent
+from cloudevents.http.event_type import is_binary, is_structured
```

Comment:
Why do we need this import?

Reply:
exporting is_binary and is_structured under a more convenient name. With that import I can now do
```python
from cloudevents.http import is_binary, is_structured
```

without I'd need to do
```python
from cloudevents.http.event_type import is_binary, is_structured
```

Reply:
Ah thanks.